platform/mds/mds/src/main/java/org/motechproject/mds/ex/object/UpdateFromTransientException.java

```diff
@@ -0,0 +1,25 @@
+package org.motechproject.mds.ex.object;
+
+import org.motechproject.mds.ex.MdsException;
+
+/**
+ * Exception, that signalizes updating object from transient state
+ */
+public class UpdateFromTransientException extends MdsException {
+
+    private static final long serialVersionUID = 3815424623777042327L;
+
+    public UpdateFromTransientException() {
+        super("When you want to update object you have to retrieve the object from database in the same transaction," +
+                "for example : " +
+                "dataService.doInTransaction(new TransactionCallbackWithoutResult() {\n" +
```

Comment:
We don't want this doInTransaction() stuff plastered across our error messages. Spring transactions is the preferred way to go